rust_dev_preview/examples/auto-scaling/README.md

```diff
@@ -0,0 +1,118 @@
+<!--Generated by WRITEME on 2023-09-08 16:28:06.995125 (UTC)-->
+# Auto Scaling code examples for the SDK for Rust
+
+## Overview
+
+Shows how to use the AWS SDK for Rust to work with Amazon EC2 Auto Scaling.
+
+<!--custom.overview.start-->
+<!--custom.overview.end-->
+
+*Auto Scaling automatically scales EC2 instances, either with scaling policies or with scheduled scaling.*
+
+## ⚠ Important
+
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+
+### Prerequisites
+
+For prerequisites, see the [README](../../README.md#Prerequisites) in the `rust_dev_preview` folder.
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+
+### Get started
+
+* [Hello Auto Scaling](None) (`DescribeAutoScalingGroups`)
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+* [Create a group](None) (`CreateAutoScalingGroup`)
+* [Delete a group](None) (`DeleteAutoScalingGroup`)
+* [Disable metrics collection for a group](None) (`DisableMetricsCollection`)
+* [Enable metrics collection for a group](None) (`EnableMetricsCollection`)
+* [Get information about groups](None) (`DescribeAutoScalingGroups`)
+* [Get information about instances](None) (`DescribeAutoScalingInstances`)
+* [Get information about scaling activities](None) (`DescribeScalingActivities`)
+* [Set the desired capacity of a group](None) (`SetDesiredCapacity`)
+* [Terminate an instance in a group](None) (`TerminateInstanceInAutoScalingGroup`)
+* [Update a group](None) (`UpdateAutoScalingGroup`)
```

Comment:
these get autofixed somehow I guess?

Reply:
Yeah, we have a WRITEME tool that will run on this before I submit